dmscripts/generate_user_email_list_applications.py

```diff
@@ -0,0 +1,110 @@
+import csv
```

Comment:
Have you seen what @TheDoubleK has done over [here](https://github.com/alphagov/digitalmarketplace-scripts/blob/master/dmscripts/insert_framework_results.py#L2). `unicodecsv` allows us to handle unicode consistently between Python 2 and 3.